### DISCUSSION TITLE
https://penguins-eggs.net/blog/build-blendos-image

### DISCUSSION THREAD

**Discussion by giscus** [pieroproietti/penguins-blog#6] [General]
# https://penguins-eggs.net/blog/build-blendos-image

I wrote something about blendOS in this previous post, now I'm trying to build blendOS image, following official blendOS Documentation and particularly this pages:

https://penguins-eggs.net/blog/build-blendos-image

**Comment by rayvermey** [comment]
Or follow this doc: https://docs.blendos.co/docs/build-blend/build_environment

**Reply by pieroproietti** [reply]
I exactly did that, but there are the problems you can find on the post: need ot install python-pip, install pip module click, then breakfast don't build system. 

I tested on blendOS in a contanier and in Arch, always the same problems.

**Comment by pieroproietti** [comment]
Today I tried again with blendOS xfce and a simply Arch system. Both have the same errors I descrived befre: lacks on python-pip, cannot find module pip click and and without producing: system
```
failed package builds:
 -> system
```

The difference is just not crying for waydroid-blend and waydroid-image not included here.

**Reply by pieroproietti** [reply]
After same other tempts, finally I get the right result! 

I started this time with a clean installation without akshara hook, and used the system directlym not in containers. 

Removing akshara hook, blendOS seem to act as normal system not as immutable.

In this way, giving the same commands i did before, and without the need to add `python-pit` and install module `click` I went until the end correctly.

Just creating a new Gnome version with `sudo brunch`

**Reply by rayvermey** [reply]
What I do is create a Directory and put the pacstrap files in it.
Then sudo arch-chroot into that and follow the build docs. 
Works like a charm.

**Reply by pieroproietti** [reply]
Thanks again @rayvermey 

I just finished to build it using the way described above... later I will try your way too. 

and...  feelme happy to be able to reproduce it!

Now I'm just trying your way, but really I'm not able to replicate.

Giving `sudo pacstrap -K .` inside the container result in:

![error-pacstrap](https://github.com/pieroproietti/penguins-blog/assets/958613/415bcf06-031d-4b71-887b-7e90c8d1157c)

**Comment by Ast3risk-ops** [comment]
Not bad, so you're trying to get penguins-eggs to work on blendOS?

**Reply by pieroproietti** [reply]
Yes, I started exactly with this idea: remaster blendOS was my way to approach it, more than think if was usefull or no. I knew from the start where was an official way to do it. 

So, when I saw yours guide, immidiatly tryed to replicate it and see that happen.

This is just the first tempt, again more to learn.
